networking/ovn_kubernetes_network_provider/ovn-kubernetes-logical-architecture.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="ovn-kubernetes-logical-architecture"]
+= OVN-Kubernetes logical architecture
+include::_attributes/common-attributes.adoc[]
+:context: ovn-kubernetes-architecture
+
+toc::[]
+
+The following diagram shows the OVN-Kubernetes logical architecture.
+
+.OVN-Kubernetes architecture
+image::ovn-k-logical-architecture.png[OVN-Kubernetes logical architecture]
+
+The key components are:
+
+* **Gateway Routers** - (sometimes called L3 Gateway Routers) are typically used between distributed routers and the physical network. It (including its logical patch ports) is bound to a physical location (not distributed), or chassis. The patch ports on this router are known as l3gateway ports in ovn-sbdb. example: GR_ovn-worker1 and GR_ovn-worker2.
+* **Distributed Logical Routers** - and the logical switches behind it, to which VMs and containers attach, effectively reside on each hypervisor. example: ovn_cluster_router.
```

Comment:
```suggestion
* **Distributed Logical Routers** - and the logical switches behind it, to which virtual machines and containers attach, effectively reside on each hypervisor. example: ovn_cluster_router.
```

Reply:
removed file as copied contents to modules